### PR TITLE
Added skeleton structure for Booking, Customer, Tour, TourGuide classes.

### PR DESCRIPTION
**Pull request by eamonngorman**
Created the basic outline of the 4 named classes with some placeholder functions.

### CODE REVIEW

File: src/main/java/com/example/bigdataproject/Tour.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    public void setTourGuide(TourGuide tourGuide) {
+        this.tourGuide = tourGuide;
+    }
+
+
+
+    // Method to check if the tour is full
+    public boolean isFull() {
+        return getTotalBookingSize() >= capacity;
```
Comment (BrenyQT) [inline src/main/java/com/example/bigdataproject/Tour.java:67]: just fix this and it lgtm

**Comment by BrenyQT** [comment]
LGTM

**Review by BrenyQT — APPROVED** [review]
LGTM

**Review by BrenyQT — COMMENTED** [review]
You could use lombok to create boiler plates instead of creating class getters/setters

File: src/main/java/com/example/bigdataproject/Customer.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    // Method to view all bookings
+    public void viewBookings() {
+        if (bookings.isEmpty()) {
+        } else {
+            // Check database for bookings
+        }
+    }
+
+    // Method to edit customer info
+    public void editInfo(String newEmail, String newName, String newNumber) {
```
Comment (carthy27) [inline src/main/java/com/example/bigdataproject/Customer.java:45]: This functionality needs to accept only one change as well as all being changed.

**Review by carthy27 — COMMENTED** [review]
Only one or two questions on how we are storing the bookings. Besides that good work

**Pull request merged**